code/__DEFINES/is_helpers.dm

```diff
@@ -228,6 +228,8 @@ GLOBAL_LIST_INIT(glass_sheet_types, typecacheof(list(
 #define isProbablyWallMounted(O) (O.pixel_x > 20 || O.pixel_x < -20 || O.pixel_y > 20 || O.pixel_y < -20)
 #define isbook(O) (is_type_in_typecache(O, GLOB.book_types))
 
+#define iscomponent(O) (istype(O, /obj/item/circuit_component))
```

Comment:
This should be `is_circuit_component`, we don't want any confusion between these and `/datum/component`.

Reply:
Convention says `iscircuitcomponent`

Reply:
don't worry about, this istypecheck is used in only one place, so I'm just getting rid of the define